ship.config.js

```diff
@@ -14,6 +14,12 @@ module.exports = {
     exec('yarn doctoc');
   },
   pullRequestTeamReviewer: ['instantsearch-for-websites'],
+  afterPublish: ({ exec }) => {
```

Comment:
We don't need to clean Git's state because it's only supposed to be run on CircleCI, right?

Reply:
Right, I didn't put code to clean it up (kind of on purpose). And if you ever run this on your computer, you'll see the change.